src/Altinn.App.PlatformServices/Implementation/AppResourcesSI.cs

```diff
@@ -437,5 +431,24 @@ private byte[] ReadFileContentsFromLegalPath(string legalPath, string filePath)
 
             return null;
         }
+        
+        /// <inheritdoc />
+        [Obsolete("GetOptions method is obsolete and will be removed in the future. Use the corresponding method in IAppOptions interface instead.", false)]
+        public List<AppOption> GetOptions(string optionId)
+        {
+            return _appOptionsService.GetOptions(optionId);
+        }
+
+        /// <inheritdoc/>
+        public async Task<AppOptions> GetOptionsAsync(string optionId, string language, Dictionary<string, string> keyValuePairs)
```

Comment:
Are these also Obsolete since `GetOptions(string)`is Obsolete? Should they even be implemented?

Reply:
No they are not obsolete as such, but they are used here as a passthrough as one step on the way to split out options related methods from IAppResources into it's own interface (IAppOptionsService). The IAppResources is to big and should be split into smaller more targeted interfaces. Established the IAppOptionsService but kept the methods in IAppResources to avoid too much refactoring and breaking changes the last time, but since we now have a breaking change and it's only PDF left that uses the methods from AppResourcesSI/IAppResources they can be switched out with a dependency to IAppOptionsService instead. 

Long story short, I'm removing them as you suggest:)